lib/msal-browser/src/interaction_client/NativeInteractionClient.ts

```diff
@@ -55,6 +56,11 @@ import { BrowserAuthError } from "../error/BrowserAuthError";
 import { SilentCacheClient } from "./SilentCacheClient";
 import { AuthenticationResult } from "../response/AuthenticationResult";
 
+const BrokerServerParamKeys = {
```

Comment:
To retain the boundary? Usually constants are define in `Constants.ts`

Reply:
These doesn't look like a constant that needs to be exported all across the module to me.

Reply:
Fair.